VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<language name="Czech" localizedName="Čeština">
+	<translation id="project">Projekt</translation>
+	<translation id="file">Soubor</translation>
+	<translation id="solution">Řešení</translation>
+
+	<translation id="noActiveProject">Žádný aktivní projekt</translation>
+	<translation id="noActiveFile">Žádný aktivní soubor</translation>
+	<translation id="noActiveSolution">Žádné aktivní řešení</translation>
+
+	<translation id="NoneAssetSlot">None</translation>
```

Comment:
The "NoneAssetSlot" key does not seem to have been translated.
Should there be a "Zádné" here?

Reply:
Yes, my bad, I'm sorry